app/assets/javascripts/covid19/src/webapp/components/structures-table/SearchBar.tsx

```diff
@@ -59,13 +59,12 @@ export const SearchBar: React.FC<SearchBarProps> = React.memo(props => {
             <div style={styles.searchBar}>
```

Comment:
Is that a problem?

![image](https://user-images.githubusercontent.com/24643/155964163-db05a7ce-e17c-4190-9d2f-065419fcd9fe.png)

Reply:
Yes, that was discussed with @adrianq but we'll figure it out depending on what the client tells us. Because we can end up with more or less chips. So if we think of a solution now, maybe we have to change it again.